### DISCUSSION TITLE
Regarding Atom Invariants

### DISCUSSION THREAD

**Discussion by manajit-das** [rdkit/rdkit#5945] [Q&A] [answered]
I wanted to know what is Invariants in a fingerprint? What is the meaning of "customAtomInvariants = [1, 1, 1]" ? How to understand it with a simple example.
Thank you.

**Comment by kozaczki** [comment, marked as answer]
have a look at : https://depth-first.com/articles/2019/01/11/extended-connectivity-fingerprints/

**Reply by manajit-das** [reply]
Thank you, I will read.
